libdnf5/base/goal.cpp

```diff
@@ -567,11 +567,27 @@ GoalProblem Goal::Impl::add_module_specs_to_goal(base::Transaction & transaction
 
     std::vector<std::string> missing_module_specs;
     for (auto & [action, spec, settings] : module_specs) {
+        bool skip_unavailable = settings.resolve_skip_unavailable(base->get_config());
+        auto log_level = skip_unavailable ? libdnf5::Logger::Level::WARNING : libdnf5::Logger::Level::ERROR;
         try {
             switch (action) {
-                case GoalAction::ENABLE:
-                    module_sack.p_impl->enable(spec);
+                case GoalAction::ENABLE: {
+                    const auto & enable_ret = module_sack.p_impl->enable(spec);
+                    if (!enable_ret.second.empty()) {
+                        transaction.p_impl->add_resolve_log(
+                            action,
+                            GoalProblem::MULTIPLE_STREAMS,
+                            GoalJobSettings(),
+                            libdnf5::transaction::TransactionItemType::MODULE,
+                            spec,
+                            enable_ret.second,
+                            log_level);
+                        if (!skip_unavailable) {
```

Comment:
Are you sure that you want to use `skip_unavailable` option for result when request is conflicting? What about `skip_broken` option.

Reply:
I have to test how code handles both situation - speck does not match any module and `GoalProblem::MULTIPLE_STREAMS`

Reply:
After testing - scenario with no match works correctly. Only what works not nicely is a case where GoalProblem::MULTIPLE_STREAMS is triggered and option --skip-unavailable ignores the issue.

Reply:
This is a minor issue, not something significant, but if it will be not addressed in the PR, then issue should be created.

Reply:
I added the `skip_broken` option.